### PR TITLE
just no

### PR DESCRIPTION
**Pull request by coredev-uk**

### CODE REVIEW

File: src/main/plugins/menubar.ts [hunk 1]
```diff
@@ -35,20 +35,33 @@ export default class Thumbar {
...
-                    click: () => utils.getWindow().webContents.executeJavaScript(`app.appRoute('settings')`),
+                    click: () => utils.getWindow().webContents.executeJavaScript(`app.appRoute('settings')`)
                 },
-                {type: 'separator'},
-                {
-                    label: utils.getLocale(utils.getStoreValue('general.language'), 'menubar.options.about'),
-                    click: () => utils.getWindow().webContents.executeJavaScript(`app.appRoute('about')`)
-                },
-                {type: 'separator'},
-                {role: 'quit', label: utils.getLocale(utils.getStoreValue('general.language'), 'menubar.options.quit')},
```
Comment (coredev-uk) [inline src/main/plugins/menubar.ts:46]: @mefsaal ima just make this clear here, the role buttons will use your OS's built in menubar items, pretty much all of them are for macos with the exception of close and a few of the view ones.

Comment (mefsaal) [reply src/main/plugins/menubar.ts:46]: Hi, well I understand. But i don't change anything in the role or type, only added:

`utils.getLocale(utils.getStoreValue('general.language')` in the “label” to make string localizated.

I don't know if this change its problematic on MacOS menubar.

Only find this in the electron page:
https://www.electronjs.org/es/docs/latest/api/menu

I will try to make menubar compatible with MacOs and Linux/OtherOS without broken the other.

Comment (coredev-uk) [reply src/main/plugins/menubar.ts:46]: The Menubar is operational again, don't worry. Just the label for the quit role is not necessary and breaks it.

Comment (mefsaal) [reply src/main/plugins/menubar.ts:46]: Hi, again. After read about MacOS Menubar and how to work with electron i found some things doesn't needed localization and duplicated in the menu.

I propose this change:  https://github.com/mefsaal/Cider/commit/32cb964647cda02506c3de51e31081d206330cb8

my idea  separate linux and mac menus option, to let mac like electron does and linux with something that give the same results

**Pull request merged**